headhunter/checkpoint.go

```diff
@@ -546,6 +549,8 @@ func (volume *volumeStruct) getEtcdCheckpointHeader() (checkpointHeader *Checkpo
 
 	checkpointHeaderEtcdRevision = getResponse.Kvs[0].ModRevision
 
+	logger.Infof("getEtcdCheckpointHeader: returning for volume '%s' etcdkey '%s' header %+v",
```

Comment:
Love these logging messages. I have half a mind to mirror them on the HEAD fetches from Swift.. but I hope we can eventually dispense with the .__checkpoint__ HEAD requests once we are assured all checkpoints are recorded in etcd.

Reply:
Sounds good.